packages/salesforcedx-vscode-core/package.json

```diff
@@ -798,11 +798,6 @@
           "default": false,
           "description": "%conflict_detection_enabled_description%"
         },
-        "salesforcedx-vscode-core.experimental.deployRetrieve": {
```

Comment:
We still need the `deployRetrieve` setting, let's not touch that :)

Reply:
Sounds good!